src/components/helpers/setPageTitleHelpers.ts

```diff
@@ -1,4 +1,4 @@
-import i18n from 'i18next';
+import i18n, { t } from 'i18next';
 
 /* 
   setPageTitle()
```

Comment:
Is this commented out line "setPageTitle()" required?

Reply:
Hi Carl it's the explanation given about setPageTitle and how it works. So no harm in keeping it

Reply:
Hi Morning, thank you for the update, this can be resolved, it does not give me an option to resolve :(